build/hellobuild.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Hello Build
+description: Build Hello World
+keywords: build, buildx, buildkit, getting started, Dockerfile, image layers, build instructions, build context
+---
+
+## Hello Build!
+
+It all starts with a Dockerfile.
+
+Dockerfiles are text files containing instructions. Dockerfiles adhere to a specific format and contain a **set of instructions** for which you can find a full reference in the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
+<br/>Docker builds images by reading the instructions from a Dockerfile.
+
+Docker images consist of **read-only layers**, each resulting from an instruction in the Dockerfile. Layers are stacked sequentially and each one is a delta representing the changes applied to the previous layer.
+
+## Dockerfile basics
+
+The Dockerfile is a text file containing all necessary instructions needed to assemble and deploy a container image with Docker.
+
+Here are the most common types of instructions:
+ 
+* [**FROM \<image\>**](https://docs.docker.com/engine/reference/builder/#from) - defines a base for your image.
```

Comment:
Could you update these to relative links?

Reply:
Done.